charts/influxdb/values.yaml

```
@@ -235,3 +235,9 @@ backup:
   #   storageAccountSecret: influxdb-backup-azure-key
   #   destination_container: influxdb-container
   #   destination_path: ""
+
+  ## Amazon S3 Bucket
+  # awsS3:
+  #   s3userKey:
+  #   s3userKeySecret:
+  #   s3bucketName: influxdb-backup
```

```suggestion
  ## Amazon S3
  # amazonS3:
  #   userKey:
  #   userKeySecret:
  #   bucketName: influxdb-backup
```

nit: Amazon S3 seems to be the name of the service as opposed to AWS S3 as per [AWS documentation](https://aws.amazon.com/s3/)

Furthermore, the keys within the amazonS3 struct don't need to specify that they are S3 as that is implied.

---

thank you @naseemkullah

---

@naseemkullah are these the changes that you wanted, or do you have anything else in your mind?

---

Please commit the suggestion if you do agree with those value name changes. This only covers the value names. As per my other comment we still need to add the logic in the backup cronjob template.